shared/teams/team/member/index.new.tsx

```diff
@@ -7,35 +7,200 @@ import * as Container from '../../../util/container'
+import {pluralize} from '../../../util/string'
+import {FloatingRolePicker} from '../../role-picker'
+import * as TeamsGen from '../../../actions/teams-gen'
 
 type Props = {
   teamID: Types.TeamID
   username: string
 }
+type OwnProps = Container.RouteProps<Props>
 
-class TeamMember extends React.Component<Container.RouteProps<Props>> {
-  static navigationOptions = (ownProps: Container.RouteProps<Props>) => ({
-    header: Container.isMobile
-      ? () => (
-          <TeamMemberHeader
-            teamID={Container.getRouteProps(ownProps, 'teamID', Types.noTeamID)}
-            username={Container.getRouteProps(ownProps, 'username', '')}
-          />
-        )
-      : undefined,
-    headerExpandable: true,
-    headerTitle: () => (
-      <TeamMemberHeader
-        teamID={Container.getRouteProps(ownProps, 'teamID', Types.noTeamID)}
-        username={Container.getRouteProps(ownProps, 'username', '')}
-      />
-    ),
+// TODO: upon visiting this page, dispatch something to load the subteam data into the store for all these subteams
+const getSubteamsInNotIn = (state: Container.TypedState, teamID: Types.TeamID, username: string) => {
+  const subteamsAll = Constants.getTeamDetails(state, teamID).subteams
+  let subteamsNotIn: Array<Types.TeamMeta> = []
+  let subteamsIn: Array<Types.TeamMeta> = []
+  subteamsAll.forEach(subteamID => {
+    const subteamDetails = Constants.getTeamDetails(state, subteamID)
+    const subteamMeta = Constants.getTeamMeta(state, subteamID)
+    const memberInSubteam = subteamDetails.members.has(username)
+    if (memberInSubteam) {
+      subteamsIn.push(subteamMeta)
+    } else {
+      subteamsNotIn.push(subteamMeta)
+    }
   })
-  render() {
-    return null
+  return {
+    subteamsIn,
+    subteamsNotIn,
   }
 }
 
+const TeamMember = (props: OwnProps) => {
+  const username = Container.getRouteProps(props, 'username', '')
+  const teamID = Container.getRouteProps(props, 'teamID', Types.noTeamID)
+  const {subteamsIn, subteamsNotIn} = Container.useSelector(state =>
+    getSubteamsInNotIn(state, teamID, username)
+  )
+  const sections = [
+    {
+      data: subteamsIn,
+      key: 'section-subteams',
+      title: `${username} is in:`,
+      renderItem: ({item, index}: {item: Types.TeamMeta; index: number}) => (
+        <SubteamInRow teamID={teamID} subteam={item} idx={index} username={username} />
+      ),
+    },
+    {
+      data: subteamsNotIn,
+      key: 'section-add-subteams',
+      title: `Add ${username} to:`,
+      renderItem: ({item, index}: {item: Types.TeamMeta; index: number}) => (
+        <SubteamNotInRow teamID={teamID} subteam={item} idx={index} username={username} />
+      ),
+    },
+  ]
+  return (
+    <Kb.SectionList
+      stickySectionHeadersEnabled={true}
+      renderSectionHeader={({section}) => <Kb.SectionDivider label={section.title} />}
+      sections={sections}
+      keyExtractor={(item: Types.TeamMeta) => `member:${username}:${item.teamname}`}
```

Comment:
I'm seeing these in the console

> Warning: Encountered two children with the same key, `member:ayoubd:undefined:`

Reply:
I didn't realize the keyExtractor also applies to the section headers, that's surprising. Fixed.